model/simulationtests/coilsystem_waterhx.rb

```diff
@@ -62,8 +62,11 @@
 plant.addDemandBranchForComponent(water_coil)
 coil.remove
 
-hx = coil_system.heatExchanger
+hx = coil_system.heatExchanger.to_HeatExchangerAirToAirSensibleAndLatent.get
 hx.setName('CoilSystemWaterHX HX')
+if Gem::Version.new(OpenStudio.openStudioVersion) >= Gem::Version.new('3.8.0')
+  hx.assignHistoricalEffectivenessCurves
+end
```

Comment:
Backward compat for HX AirToAir Sensible and Latent.

OS aligned the default on E+, which means constant efficiency (and no need for Table:Lookup objects). But to maintain historical EUI, need to use the convenience function to set the efficiency at 75% like it was pre 24.1.0